user-guide/Advanced_Functionality/DataMiner_Agents/Exporting_and_importing/Exporting_elements_services_etc_to_a_dmimport_file.md

```diff
@@ -19,7 +19,9 @@ In the DataMiner Cube Surveyor, you can right-click a view in order to export it
 1. Clear the selection from any items you do not want to include in the export.
 
    > [!NOTE]
-   > If you have selected to include a redundancy group, any elements within that group will automatically be included in the export. Similarly, if you have selected an SLA, the SLA service and its service children will automatically be included.
+   >
+   > - If you have selected to include a redundancy group, any elements within that group will automatically be included in the export. Similarly, if you have selected an SLA, the SLA service and its service children will automatically be included.
+   > - From DataMiner 10.4.9/10.5.0 onwards<!--RN 39356-->, if you have selected to include SNMPv3 elements, importing this package on DataMiner Systems prior to DataMiner 10.4.9/10.5.0 will result in the loss of all SNMPv3 credentials, which will need to be re-entered manually. If you used the [DataMiner Cube Credentials Library](xref:Managing_predefined_sets_of_credentials_for_SNMP_authentication) to configure the SNMPv3 credentials, you will always need to re-enter the credentials manually after an export, regardless of your DataMiner version.
```

Comment:
I've added information about the Credentials Library. Is this correct? @33sGH

Reply:
[DataMiner Cube Credentials Library](xref:Managing_predefined_sets_of_credentials_for_SNMP_authentication) to configure the SNMPv3 credentials, you will always need to re-enter the credentials manually after an **import on a different DMS**, regardless of your DataMiner version.

changed "export" to " import on a different DMS". Because if you would import on a different agent of the same DMS where the element was exported from, it will have the credentials because the credentials library is shared in the DMS.

Reply:
```suggestion
   > - From DataMiner 10.4.9/10.5.0 onwards<!--RN 39356-->, if you have selected to include SNMPv3 elements, importing this package on DataMiner Systems prior to DataMiner 10.4.9/10.5.0 will result in the loss of all SNMPv3 credentials, which will need to be re-entered manually. If you used the [DataMiner Cube Credentials Library](xref:Managing_predefined_sets_of_credentials_for_SNMP_authentication) to configure the SNMPv3 credentials, you will always need to re-enter the credentials manually after an import on a different DMS, regardless of your DataMiner version.
```